portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,17 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Alec!");
+    ArrayList<String> myarraylist = new ArrayList<String>();
+    myarraylist.add("One");
+    myarraylist.add("Two");
+    myarraylist.add("Three");
+   
+    Gson gson = new Gson();
+    String json = gson.toJson(myarraylist);
+    response.setContentType("application/json;");
```

Comment:
Not sure how it worked, but I think there should be no semicolon (`;`) at the end of the `setContentType()` function's parameter string. Please fix this by removing the semicolon in the next PR (though you'll be changing the whole line to something else).